tree/ntuple/v7/inc/ROOT/RDaos.hxx

```diff
@@ -88,8 +128,8 @@ class RDaosObject {
 private:
    daos_handle_t fObjectHandle;
 public:
-   using DistributionKey_t = std::uint64_t;
-   using AttributeKey_t = std::uint64_t;
+   using DistributionKey_t = RDaosKey::DistributionKey_t;
+   using AttributeKey_t = RDaosKey::AttributeKey_t;
```

Comment:
Given that `RDaosObject` internally uses `DAOS_OT_MULTI_UINT64` (i.e., multi-level KV with uint64 [ad]keys; see [here](https://github.com/root-project/root/blob/bec517cb5b474c82513f10f813aa111c6e678a1b/tree/ntuple/v7/src/RDaos.cxx#L109)), I think it's slightly more correct for this to be `using XxxKey_t = std::uint64_t`.

Thus, it's `RDaosKey::XxxKey_t` that should be an alias to this type instead.